packages/shared/components/molecules/SelectorInput.svelte

```diff
@@ -108,7 +108,7 @@
                             </Text>
                         </div>
                         {#if option.value && option.key}
-                            <Text type={TextType.pre} fontSize="sm" color="gray-600" classes="flex-1">
+                            <Text type={TextType.pre} fontSize="sm" color="gray-600" classes="flex-1 text-end">
```

Comment:
What is the reason for adding text-end here?

Reply:
Without `text-end`
![image](https://github.com/iotaledger/firefly/assets/105642810/d5b1b900-e1a5-4113-b4c8-b94545d8646d)

With `text-end`
![image](https://github.com/iotaledger/firefly/assets/105642810/db4e1da3-8d49-496c-8aea-f56a01507e76)